### ISSUE TITLE
feat(quorum): run tx endpoint to use contract ID from consortium model

### ISSUE THREAD

**Issue by petermetz** [hyperledger/cactus#373]
**Is your feature request related to a problem? Please describe.**

Depends on #371 

If and when we introduce contracts being represented in the consortium model (as per #371) then we'll need to make it possible to invoke contracts by specifying the contract ID for the Quorum connector rather than the entire ABI.

**Describe the solution you'd like**

Something that still allows completely flexibility for contracts that are unregistered[1].

[1]: Not explicitly present in the consortium definition.

**Describe alternatives you've considered**

Nothing, if we go through with #371 then we'll definitely need this as a consequence.

**Additional context**

This issue is basically the same as #372 which is for Besu.

cc: @takeutak @sfuji822 @jonathan-m-hamilton

**Comment by RafaelAPB** [comment]
Closing this as it is not needed for now
